project_name/settings_production.py

```diff
@@ -1,30 +1,29 @@
-# flake8: noqa
-from {{project_name}}.settings_shared import *
+from {{project_name}}.settings_shared import *  # noqa: E999, F403
```

Comment:
@nikolas E999 is for syntax errors, do we really want that? I don't think the codes matter once you add the `# noqa`, so maybe it's not important.

Reply:
you're right, I just assumed this was necessary in order to pass this repository's flake8... if it works without that ignore then that works too.